EPI/parse-nested-GrFN.py

```diff
@@ -0,0 +1,124 @@
+import json
+from argparse import ArgumentParser
+
+
+def formatGraph(data): 
+    modelMetadata = {}
+    nodesDict = {}
+    nodes = []
+    edges = []
+
+    for key,value in data.items():
```

Comment:
Should we iterate over all items in the json, and check for the fields we're interested in? What if new fields are added in the future. Then we're iterating over all fields unnecessarily. Not a big deal, for sure. But it might be more clear to do something like `variables = data['variables']` and then `for variable in variables:`, instead of having separate if-statements.

Reply:
I'll make a note on this and address this in a separate PR in conjunction with the CAG parser. Thank you!